code/modules/client/preferences.dm

```diff
@@ -1566,6 +1566,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	return 1
 
 /datum/preferences/proc/copy_to(mob/living/carbon/human/character)
+	character.change_species(species) // Yell at me if this causes everything to melt

```

Comment:
`set_species` is used far more often than `change_species`. It might be safer to go with that instead.